ui/src/main/java/org/eclipse/starter/ui/Project.java

```diff
@@ -302,25 +311,33 @@ public void generate() {
 						entry("jakartaVersion",
 								((jakartaVersion % 1.0 != 0) ? String.format("%s", jakartaVersion)
 										: String.format("%.0f", jakartaVersion))),
-						entry("profile", profile), entry("javaVersion", javaVersion),
-						entry("docker", (docker ? "yes" : "no")), entry("runtime", runtime)));
+						entry("profile", profile),
+						entry("javaVersion", javaVersion),
+						entry("docker", (docker ? "yes" : "no")),
+						entry("runtime", runtime),
+						entry("groupId", groupId),
+						entry("artifactId", artifactId),
+						entry("package", groupId)));
 
 				MavenUtility.invokeMavenArchetype("org.eclipse.starter", "jakarta-starter", VersionInfo.ARCHETYPE_VERSION,
 						properties, workingDirectory);
 
 				LOGGER.info("Creating zip file.");
-				ZipUtility.zipDirectory(new File(workingDirectory, "jakartaee-hello-world"), workingDirectory);
```

Comment:
This is not at all a small change. To go along with the file name change here, many other things in the project such as the documentation, generated README, generated Dockerfile, etc, must change. This PR must fully address all those changes. Otherwise, this PR seriously compromised project stability. Please review the project carefully and identify all changes required.

Reply:
The problem is that it creates a further usability hazard by highlighting the fact that in the initial versions we really did not prioritize fully implementing the artifact ID and group ID, including at the Archetype level. So if we go this route now, let’s kindly implement it all the way. 

Otherwise we are opening up a situation where we will need to implement it anyway, just at a more inconvenient time. A more comprehensive approach avoids this by simply taking the time up front to do it thoroughly in the first place instead of piecemeal - which I don’t know that I have the bandwidth for. I’d rather just review a comprehensive PR once.